Kfilter/src/main/java/com/isaacudy/kfilter/KfilterView.kt

```diff
@@ -66,7 +67,8 @@ class KfilterView @JvmOverloads constructor(context: Context,
     private var surfaceHeight: Int = 0
 
     private var isPlayingOnDetach = false
-
+    var selectedKfilterStart = 0
+    var isChange: Boolean = false
```

Comment:
What is the purpose of the `isChange` variable?

Reply:
it checks filter has changed or not

Reply:
Can we rename to "isFilterChanged"?